Assets/Scripts/Achievement/AchievementManager.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using BTScriptableObject;
+using Common;
+using System;
+using UI;
+
+namespace AchievementM
+{
+    public class AchievementManager : Singleton<AchievementManager>
+    {
+        [SerializeField]
+        private AchievementScriptable achievementScriptable;
+
+        public AchievementController hiScoreController { get; private set; }
```

Comment:
can be private no need to expose outside

Reply:
Done